### PR TITLE
Bug 1282441 - skip file when s3 key is missing

### PR DESCRIPTION
**Pull request by maurodoglio**

### CODE REVIEW

File: moztelemetry/spark.py [hunk 1]
```diff
@@ -413,6 +417,13 @@ def _read_v2(filename):
 def _read_v4(filename):
     try:
         key = _bucket_v4.get_key(filename)
+
+        if key is None:
+            # In some rare cases it's not possible to retrieve the content
+            # of a file. Let's log the problem and move on.
```
Comment (mreid-moz) [inline moztelemetry/spark.py]: Can you add a comment "see bug 1282441 for more details"?

**Comment by vitillo** [comment]
Looks good, thanks! You have to increase the version number as well though.

**Comment by vitillo** [comment]
Please redeploy on pypi.

**Comment by maurodoglio** [comment]
https://pypi.python.org/pypi/python_moztelemetry/0.3.9.12

**Pull request merged**